polaris.shopify.com/content/foundations/patterns/error-messages.md

```diff
@@ -805,44 +521,27 @@ Don’t use when:
   doesn’t immediately appear in context. In these cases, use the
   [page-level banner](#page-level-banners)
 
-### Component:
+### Component
+
+- [Banner](/components/banner#navigation)
+
+Variant
```

Comment:
Missing a colon after this one

Reply:
Further down in this file, you use proper `### Headings` for these. Maybe use headings consistently for "Variant" and "Content"?